docs/cryptodoc/src/07_rng.rst

```diff
@@ -1054,5 +1060,29 @@ It works as follows:
    4. if (:math:`min \leq r < max`) return ``r``.
    5. Go to Step 3.
 
-**Conclusion:** The algorithm is a slight adaptation of Algorithm B.1 in Section B.4 of [TR-02102-1]_
+**Note:** Technically, the base implementation above should be defined for exactly :math:`min = 0`
+not :math:`min \in \{0,1\}`. When `adding this special case in Botan 3.1.0 <https://github.com/randombit/botan/pull/3594>`_
+this was done to avoid breaking existing test cases that depended on the determinism of this method for :math:`min = 1`.
+
+.. admonition:: ``BigInt::random_integer()`` (for :math:`min > 1`)
+
+   **Input:**
+
+   -  ``rng``: random number generator (see :ref:`rng/main` for available algorithms)
+   -  ``min``: integer lower bound of desired range (with :math:`min > 1`)
```

Comment:
I think we should stay consistent with writing _min_, _max_, and _r_ in code style or in math style. (Not only here but in this whole section)